compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/TextFieldSelectionManager.kt

```diff
@@ -311,6 +311,9 @@ internal class TextFieldSelectionManager(
                     isStartHandle = false,
                     adjustment = adjustment
                 )
+
+                undoManager?.forceNextSnapshot()
```

Comment:
Do we have the same behaviour in android? Then, It is better to create an issue in Google tracker and upstream this change to aosp

Reply:
Since this issue is related to a cursor, we need someone with real android device to check it after connecting the physical mouse. I didn't find a way to mimic hardware mouse on Android emulator.

Reply:
@igordmn said it's not reproducible on Android

Reply:
From my point of view, if something is not reproducible on Android, then the issue is not in the common code, but how we use it. Since, if there is no error in Android, it means that we won't easily upstream it to androidx, and we will have to keep this change only in out common code.
@igordmn WDYT?

Reply:
Undo/Redo and text selection is written in common code by us, and it can be used only partially in Android.

So, even if it is not reproducible in Android, the issue can still be in common code, not in desktop/skiko code.

Reply:
Ok, then let's keep do this change and upstream it to androidx